python/docs/source/reference/pyspark.pandas/general_functions.rst

```diff
@@ -41,6 +41,7 @@ Data manipulations and SQL
 
    melt
    merge
+   merge_asof
```

Comment:
We added this from https://github.com/apache/spark/pull/34053, but missing in our doc so far.